debian/neobotix-tools/opt/neobotix/tools/cleanup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+if [ $SUDO_USER ]; then user=$SUDO_USER; else user=`whoami`; fi
+
+# check for root privileges
+uid=$(/usr/bin/id -u) && [ "$uid" = "0" ] || { echo "Run this script as root!"; exit 1; }
+
+# empty trash
+rm -rf /home/$user/.local/share/Trash/*
+
+# remove unused packages
+apt-get -y autoremove
+
+# delete ros logs
+rm -rf /home/$user/.ros/log/*
+
+# delete pilot logs
+rm -rf /home/$user/pilot/user/data/logs/*
+
+# delete system logs
+rm -rf /var/log/*
+
+# this is needed for journalctl (systemd)
+mkdir /var/log/journal
+
+# clear bash history
+history -c
```

Comment:
unfortunately :frowning_face: , history needs to be cleared as just user.. else it complains:

`./cleanup.sh: 29: history: not found`

Reply:
I did some tests. I was not able to figure out a way to clear current history of the parent session.

Reply:
the easiest way for this right now would be to remove the `~/.bash_history` found in the home directory. I guess you can do this even with the `sudo command`

`history -c` just removes the history only on the terminal from which we are running the command. But the history would pop again if we open a new terminal once again..